src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.hpp

```diff
@@ -542,19 +535,16 @@ class ShenandoahCardCluster: public CHeapObj<mtGC> {
     while (card_index <= last_card_index)
       object_starts[card_index++] = 0;
   }
-#endif  // FAST_REMEMBERED_SET_SCANNING
 
   ShenandoahCardCluster(RememberedSet *rs) {
     _rs = rs;
-#ifdef FAST_REMEMBERED_SET_SCANNING
-    // HEY!  We don't really need object_starts entries for every card entry.  We only need these for the
+    // HEY!  We don't really need object_starts entries for every card entry.  We only need these for
```

Comment:
Consider changing 'HEY!' to 'TODO' or something. IDEs tend to highlight the latter (mine does).

Reply:
I'll replace these en masse with another pull request.